working-together/README.md

```diff
@@ -0,0 +1,23 @@
+Working Together
+==============
+
+A guide for working well together.
+
+
+Pair Programming
+-------
+
+**Before:**
```

Comment:
I've run into pairing environments where it's tough to pair due to limited desk space. It feels awkward to bring it up and ask the person to move their stuff off the desk. As part of the **before**,  can we include:
- Agree on a location (Your desk, mine, or another location? Create space to accommodate two people and two keyboards.)

Reply:
Really good point!